### ISSUE TITLE
Passing to vue

### ISSUE THREAD

**Issue by randohinn** [spatie/sheets#19]
I need to pass a Sheet to Vue. Doing the standard 

```php
 @foreach($sheets as $work)
            <workdot :work="{{$work}}"></workdot>
@endforeach
```

Ends up with a `htmlspecialchars() expects parameter 1 to be string, object given`. Seems like you have not thought of this use-case and the sheet can't be converted to json?

**Comment by sebastiandedeyne** [comment]
Fixed in v1.2.0!

**Comment by randohinn** [comment]
That was fast! Thanks :D!
